src/components/IconButtonBar/IconButtonBar.js

```diff
@@ -73,6 +74,7 @@ const IconButtonBar = ({
         key={action.label}
         onClick={action.onClick}
         disabled={action.disabled}
+        {...action.otherProps}
```

Comment:
Is there an opportunity to spread these flat from the `action` as opposed to having them contained within an `otherProps` property? I'm unfamiliar with this pattern if it's in use elsewhere

Reply:
Consider that the `selectorPrimaryFocus` targets a selector in a child `OverflowMenuItem`, which can be anything the user wants.